infra/environments/prod/terraform.tfvars

```diff
@@ -3,3 +3,7 @@ role_arn = "arn:aws:iam::245339368379:role/terraform_deploy_role"
 image_name    = "hello_world"
 data_bucket   = "test-data"
 function_name = "hello-lambda"
+
+# downloads/dns
+download_bucket = "downloads.curibio.com"
```

Comment:
out of curiousity, is there a way to say something like `"downloads." + hosted_zone` is the name of the bucket, instead of having a separate variable?

Reply:
changed so it does string interpolation with the subdomain + hosted_zone values